src/julia.expmap

```diff
@@ -32,7 +32,7 @@
     utf8proc_*;
     jlbacktrace;
     _IO_stdin_used;
-    llvm_to_jl_value;
+    __ZN4llvm23createLowerSimdLoopPassEv;
```

Comment:
you really shouldn't be exporting C++ clang-mangled names. the C ABI exists for a reason.